src/lib/libraries/decks/index.jsx

```diff
@@ -1068,119 +1071,106 @@ export default {
         ],
         urlId: 'pong'
     },
-
-    'say-it-out-loud': {
+    
+    'cartoon-network': {
         name: (
             <FormattedMessage
-                defaultMessage="Create Animations That Talk"
-                description="Name for the 'Create Animations That Talk' how-to"
-                id="gui.howtos.say-it-out-loud"
+                defaultMessage="Animate an Adventure Game"
+                description="Animate an Adventure Game' how-to"
+                id="gui.howtos.cartoon-network"
             />
         ),
-        img: libraryTXTSpeech,
+        requiredProjectId: '249143200',
+        img: libraryCartoonNetwork,
         steps: [{
-            video: 'k54n8uwcty',
+            video: 'uz5oz5h9yg',
             trackingPixel: (
-                <img src="https://code.org/api/hour/begin_scratch_talk.png" />
+                <img src="https://code.org/api/hour/begin_scratch_adventure.png" />
```

Comment:
Again, what's the use for it?

Reply:
This line appears to be changed, but it's actually just a re-ordering of the content in the file. So we are not adding, removing, or changing anything related to tracking pixels. The tracking pixels were used last year by code.org (the organization that operates hour of code each year) for their analytics. They are no longer used. We use a different system for our own analytics.

Reply:
If it's no longer used, it should be removed.